packages/config/config/devices/0x027a/zen23_4.0.json

```
@@ -0,0 +1,238 @@
+// Zooz ZEN23
+// Z-Wave Plus On/Off Toggle Switch v4
+{
+	"revision": 1,
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN23",
+	"description": "Z-Wave Plus On/Off Toggle Switch v4",
+	"devices": [
+		{
+			"productType": "0xb111",
+			"productId": "0x251c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "4.0",
+		"max": "4.255"
+	},
+	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Group 2",
+			"description": "BASIC Command Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Toggle Control",
```

How about this (also see next suggestion):
```suggestion
			"label": "Switch behavior",
```

---

I was a little worried about confusing going with `Switch behavior`, because technically there's both the physical switch and the internal light control switch. But it's probably not that confusing to someone fiddling with the settings.